### ISSUE TITLE
Latest version does not have latest changes.

### ISSUE THREAD

**Issue by statop** [fluttercommunity/flutter_sms#75]
It seems that the latest version on pub.dev (2.3.3) doesn't have the latest changes.

for example, this changes is not in 2.3.3 - https://github.com/fluttercommunity/flutter_sms/commit/947c96d29b98ab3db45287e3e5f7740f27cfa3d0

**Comment by AliPunjabi** [comment]
> It seems that the latest version on pub.dev (2.3.3) doesn't have the latest changes.
> 
> for example, this changes is not in 2.3.3 - [947c96d](https://github.com/fluttercommunity/flutter_sms/commit/947c96d29b98ab3db45287e3e5f7740f27cfa3d0)

yeah and it also doesn't support kotlin version above than 1.3.50 or 1.4.20 maybe not sure, but some other dependencies like flutter local notification needs higher gradle version which depends on kotlin version above than the mentioned above for this dependency. So it's a major conflict as if we need to show notifications on android 13, we can't use the older version of flutter local notifications and for this we need to use the higher version of gradle.
